.evergreen/download-mongodb.sh

```diff
@@ -49,10 +49,10 @@ get_mongodb_download_url_for ()
    _DISTRO=$1
    _VERSION=$2
 
-   VERSION_42="4.2.2"
-   VERSION_40="4.0.13"
-   VERSION_36="3.6.15"
-   VERSION_34="3.4.23"
+   VERSION_42="4.2.3"
+   VERSION_40="4.0.16"
+   VERSION_36="3.6.17"
+   VERSION_34="3.4.24"
    VERSION_32="3.2.22"
```

Comment:
Not sure why this showing up on my PR but it's from this commit: https://github.com/mongodb-labs/drivers-evergreen-tools/commit/84df805153108979986653036e9836cd578eaeca

Reply:
Perhaps it will go away if you rebase/force push this branch?

Reply:
Done